tiatoolbox/utils/transforms.py

```diff
@@ -42,6 +42,9 @@ def background_composite(
         >>> plt.show()
 
     """
+    if image.size[:, :, 0] >= 4:
```

Comment:
This will fail for RGBA image. I think the best way to handle this is using mode https://github.com/TissueImageAnalytics/tiatoolbox/issues/609